proposals/841-release-process/README.md

```diff
@@ -0,0 +1,115 @@
+# KEP-841: Proposal to update the Release Process for Kubeflow
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+## Summary
+
+This proposal aims to share insights on topics discussed with the release team and others about diverse challenges and improvement opportunities in the release process based on Kubeflow release 1.10.
+
+## Motivation
+
+Current challenges based on Kubeflow release 1.10:
+* Challenge to get component’s release progress during release meetings.
+    - Blockers or challenges on the Release are identified later.
+    -  Incomplete/Inaccurate or null information on some components about progress/blockers.
+* The Release Manager role challenges:
+- Only one person is responsible for the Release and decisions.
+- Time dedication and commitment are high and for an extended period (6 months), which makes it hard to find candidates for Release.
+- The diverse skills required for the role make it even harder for candidates to join.
+* Release Documentation
+- Working groups need help on building Technical Documentation for Releases.
+- Working on technical documentation during the Release, before the feature freeze, can expedite the release process.
+* Feature Freeze vs Building Release Candidate
+- Teams required more time to finish the Release and later cherry-pick and work on building the Release.
+* Release timeframe 
+- The Release is now 27 weeks. There are 10 weeks between feature freeze and Release. 
+- Release dates and phases are hard to predict in the current process.
+
+### Goals
+
+* Reduce time to market of Kubeflow Release
```

Comment:
```suggestion
* Reduce the process and overhead and human hours needed
* Move away from the Waterfall model to an agile model that relies on automated tests instead of human labor.
* Reduce time to market of Kubeflow Release
```